opentracing-api/src/main/java/io/opentracing/tag/Tags.java

```diff
@@ -36,6 +36,20 @@ private Tags() {
     public static final String SPAN_KIND_CLIENT = "client";
 
     /**
+     * A constant for setting the span kind to indicate that it represents a producer span, in a messaging scenario.
+     *
+     * @see <a href="https://github.com/opentracing/specification/blob/master/semantic_conventions.md#message-bus">https://github.com/opentracing/specification/blob/master/semantic_conventions.md#message-bus</a>
```

Comment:
it's going to be tedious keeping these links up to date. I think the single link at the top of the file is enough.